tests/integrations/mcs/scheduling/api_test.go

```diff
@@ -159,10 +159,17 @@ func (suite *apiTestSuite) TestAPIForward() {
 	pauseArgs, err := json.Marshal(input)
 	suite.NoError(err)
 	err = testutil.CheckPostJSON(testDialClient, fmt.Sprintf("%s/%s", urlPrefix, "checker/merge"), pauseArgs,
-		testutil.StatusOK(re), testutil.WithoutHeader(re, apiutil.PDRedirectorHeader))
```

Comment:
Why change it?

Reply:
Because we should allow to redirect pause/resume checker